scapy/packet.py

```diff
@@ -1390,7 +1389,7 @@ def _do_summary(self):
 
     def summary(self, intern=0):
         """Prints a one line summary of a packet."""
-        found, s, needed = self._do_summary()
+        _, s, _ = self._do_summary()
```

Comment:
I do not agree with that change (no matter what the linter says). Using `_` is a common way to show that you don't need the results. In this specific case however, you could use directly:
```Python
         return self._do_summary()[1]
```